www/include/options/session/connected_user.php

```diff
@@ -122,7 +122,7 @@
     if ($centreon->user->admin) {
         // adding the link to be able to kick the user
         $session_data[$cpt]["actions"] =
-            "<a href='./main.php?p=" . $p . "&o=k&session=" . $r['session_id'] . "'>" .
+            "<a href='./main.php?p=" . $p . "&o=k&user=" . $r['user_id'] . "'>" .
```

Comment:
Kindly, could you correct the line 142 too : 
https://github.com/centreon/centreon/pull/8876/files#diff-576736f130763acc5da4139d11faa7beR142

Reply:
but you may need to check the JS called

Reply:
Fixed with #2386868